test/jasmine/tests/transform_expanded_trace_test.js

```diff
@@ -0,0 +1,30 @@
+var Plots = require('@src/plots/plots');
```

Comment:
No need for a new file here.

Can you move this to either `transform_groupby_test.js` or the poorly-named `transform_multi_test.js`?